Source/Game/Game/Editor/CameraInfo.cpp

```diff
@@ -107,9 +107,9 @@ namespace Editor
 
             ImGui::Separator();
 
-            ImGui::Text("Forward: (%.2f, %.2f, %.2f)", cameraTransform.forward.x, cameraTransform.forward.y, cameraTransform.forward.z);
-            ImGui::Text("Right: (%.2f, %.2f, %.2f)", cameraTransform.right.x, cameraTransform.right.y, cameraTransform.right.z);
-            ImGui::Text("Up: (%.2f, %.2f, %.2f)", cameraTransform.up.x, cameraTransform.up.y, cameraTransform.up.z);
+            ImGui::Text("Forward: (%.2f, %.2f, %.2f)", cameraTransform.GetLocalForward().x, cameraTransform.GetLocalForward().y, cameraTransform.GetLocalForward().z);
```

Comment:
Isn't his a matrix multiplication everytime we do GetLocalX, maybe we should GetLocalForward once and then reuse it printing x, y and z? Same for the next two lines